### ISSUE TITLE
change "/de" endpoint to "/add"

### ISSUE THREAD

**Issue by readytheory** [regularstuff/tildeblog#33]

**Comment by readytheory** [comment]
Since merge of issue #20, create and update share logic.
Even though they use same logic, we should have one endpoint /add  and one /edit/123 (to edit item 123)

**Comment by Akhaled19** [comment]
Hello @readytheory! If I understand correctly, you're requesting the following paths to be changed to this format, correct?
```Python 
urlpatterns = [
  ...
  path("edit/<int:learnt_id>", views.learning_data_entry, name="til_edit_id"),
  path("add", views.learning_data_entry, name="til_add"),
]
```

**Comment by readytheory** [comment]
@Akhaled19  Yes, that sounds like that sums it up.   Would you like to create a pull request for that?

**Comment by Akhaled19** [comment]
@readytheory great! I'll go ahead and make a pr

**Comment by regularstuff** [comment]
I missed it when you asked before, but the "name" attribute should not change, unless the places in the code that use the names "til_edit_de" and "til_de" also change.  I put a comment on PR, let me know if unclear.

If you run the site on your local, you'll see the "reverse match" failures when submitting/following links with the proposed change.
